nautobot/extras/api/views.py

```diff
@@ -349,13 +349,14 @@ def _create_schedule(serializer, data, commit, job, approval_required, request):
         name=name,
         task="nautobot.extras.jobs.scheduled_job_handler",
         job_class=job.class_path,
+        job_model=job_model,
```

Comment:
This addresses another bug that if `job_model` isn't provided the job approval in the UI will fail with "There is no job associated with this request".

Reply:
Very curious if we could clean this up a little bit in the future.

Since `job` is derived from `job_model`, we should be able to obtain that here.

Unless we are looking to leverage the existing instantiation of `job_class()` from `_run_job`. I understand `job_class` is the job's source code, but my thought is around the careful handling to ensure `job.class_path` is the same as `job_model.class_path`. This is all internal so shouldn't have too much user-input concern but might be a worthwhile safety check at the least.